### DISCUSSION TITLE
custom domain

### DISCUSSION THREAD

**Discussion by CodeCraftsman20** [community/community#142332] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can I use a custom domain with GitHub Pages?

**Comment by CodewithShahriar** [comment]
Yes, you can! To use a custom domain, follow these steps:

1.Add your custom domain in the GitHub Pages section of your repository settings.
2.Create a CNAME record in your domain's DNS settings pointing to username.github.io.
3.Once configured, your site will be accessible via your custom domain!

**Reply by CodeCraftsman20** [reply]
Thank u

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142329#discussioncomment-11022784) for reference to why this discussion was closed and locked. Thanks!
